docs/configuration.asciidoc

```diff
@@ -455,7 +455,7 @@ See https://www.owasp.org/index.php/Information_exposure_through_query_strings_i
 [[config-global-labels]]
 ==== `GlobalLabels` (added[1.2])
 
-Labels added to all events, with the format `key=value[,key=value[,...]]`.
+Labels added to all events, with the format `"GlobalLabels": "key=value,key=value,..."`.
```

Comment:
Something like

```suggestion
Labels added to all events, where each label is specified as `key=value` pair in which the `key` is the label name and `value` is the label value, and where multiple labels are each separated by a comma, for example, `label1=value1,label2=value2`.
```

@bmorelli25 might be able to come up with something more succinct.

Reply:
I think that's fine. I would probably separate the thought into a few different sentences to make it easier to consume:

> Labels added to all events. Labels are `key=value` pairs, where `key` is the label name and `value` is the label value. Separate multiple labels with a comma. For example, `label1=value1,label2=value2`.